modules/sdk/node-build/configuration/getGlobalSymbols.mjs

```diff
@@ -0,0 +1,20 @@
+import path from 'path';
+
+import getRootDir from '../util/getRootDir.mjs';
+import require from '../util/require.mjs';
+
+/**
+ * @returns
+ * Something like:
+ *
+ * {
+ *   '@clayui/charts': ['__esModule', 'bb', 'default']
+ * }
+ */
+export default async function getGlobalSymbols() {
```

Comment:
This already existed in npm-scripts (eg: https://github.com/liferay/liferay-portal/blob/04c0268a74a72620e8a6640f9bcbb877e644775d/modules/apps/frontend-js/frontend-js-dependencies-web/npmscripts.config.js#L32-L36). 

It's only that I've promoted it to global configuration level because it may need to be accessed from consumer modules too and because (to me) it makes more sense to centralize it, TBH.

Reply:
> in what case would I need to add one to the root config

Same as npm-scripts: when any package fails at runtime due to a wrong autodetection of the symbols it exports.